pyproject.toml

```diff
@@ -47,7 +47,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
-fastapi = ">=0.78,<0.90"
+fastapi = ">=0.78,<1.0.0"
```

Comment:
I prefer managing the dependency upgrades in individual PRs, can you remove the dependency upgrades?